sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/CosmosSchedulers.java

```diff
@@ -27,4 +29,22 @@
         TTL_FOR_SCHEDULER_WORKER_IN_SECONDS,
         true
     );
+
+    // Custom bounded elastic scheduler to switch off IO thread to process response.
+    public final static Scheduler BULK_EXECUTOR_BOUNDED_ELASTIC = Schedulers.newBoundedElastic(
+        2 * Schedulers.DEFAULT_BOUNDED_ELASTIC_SIZE,
+        Schedulers.DEFAULT_BOUNDED_ELASTIC_QUEUESIZE,
+        BULK_EXECUTOR_BOUNDED_ELASTIC_THREAD_NAME,
+        TTL_FOR_SCHEDULER_WORKER_IN_SECONDS,
+        true
+    );
+
+    // Custom bounded elastic scheduler to switch off IO thread to process response.
+    public final static Scheduler BULK_WRITER_BOUNDED_ELASTIC = Schedulers.newBoundedElastic(
```

Comment:
This is only used in Spark, should it be defined up there?

Reply:
Fair - will change it